rust/Cargo.toml

```diff
@@ -2,9 +2,14 @@
 name = "wasi-nn"
 version = "0.1.0"
 authors = ["The Bytecode Alliance Developers"]
+description = "High-level Rust bindings for WASI-NN"
```

Comment:
Let's go with the lowercase for consistency... I've used `wasi-nn` everywhere else.